weaviate/gql/get.py

```diff
@@ -52,14 +52,26 @@ def __str__(self) -> str:
         return "hybrid:{" + ret + "}"
 
 
+@dataclass
+class ReferenceProperty:
+    property_name: str
+    in_class: str
```

Comment:
@dirkkul we use `class_name` in other places. What about just staying with `class_name` for consistency?

Reply:
I have a feeling that the names mean slightly different things and are easy to confuse:

- `property_name` refers to the current class
- `in_class` to the Class the reference points to, so I think the parameter name should somehow reflect that.
- `properties` refers to the properties of the referenced class, I feel like this is too similar to `property_name`, but can't think of anything that is not superlong